_includes/commands/hr-cmds.md

```diff
@@ -0,0 +1,10 @@
+
+| Command              | Description                                                                           |
+| -------------------- | ------------------------------------------------------------------------------------- |
+| InsertHorizontalRule | Inserts a horizontal rule at the cursor location or inplace of the current selection. |
```

Comment:
This is actually a core command, not a command for the `hr` plugin.

Reply:
moved